### DISCUSSION TITLE
About kyverno "add-safe-to-evict-to-pods" cluster policy

### DISCUSSION THREAD

**Discussion by harsha1163** [deployKF/deployKF#176] [Questions & Answers]
I just want to understand about the kyverno cluster policies that are created by deploykf as part of manifest generation before apply to cluster.

Is deploykf responsible for creating a cluster policy named "add-safe-to-evict-to-pods" ?

I verified all the files which are generated by deploykf. but nothing found. Can anyone help me? I want to know if the policy is being created by deploykf. if yes, help me in navigating to the line in values/generator files responsible for this....

**Comment by thesuperzapper** [comment]
@harsha1163 deployKF does not include any Kyverno `ClusterPolicy` called `add-safe-to-evict-to-pods`.

Someone else might be adding it by setting one of the `xxxx.xxxx.extraManifests` values, but that would be easy to check. If you are not sure what created this resource, you might want to look at its YAML and see if it is owned by any ArgoCD applications (check labels), or was created manually.
